### PR TITLE
Add zeebeCfg variable for configmap

### LINKED ISSUE #14: Expose Zeebe configuration map
Would it be possible to define a configuration map on helm installation? Honestly I'm not really familiar with the helm charts.

So for example what we often use is a config like this:

```
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: zeebe
  labels:
    app: zeebe
data:
  zeebe.cfg.toml: |
    [cluster]
    partitionsCount = 3
    replicationFactor = 3

    [threads]
    cpuThreadCount = 4
    ioThreadCount = 4

    [gateway.cluster]
    requestTimeout = "60s"

    [gateway.monitoring]
    enabled = true

    [data]
    snapshotPeriod = "15m"
    maxSnapshots = "3"

    [[exporters]]
    id = "elasticsearch"
    className = "io.zeebe.exporter.ElasticsearchExporter"

    [exporters.args]
    url = "http://elasticsearch-master-headless:9200"
```

### PR DESCRIPTION
**Pull request by Zelldon**
In order to be more dynamic with the zeebe.cfg.toml and the configmap.
 The zeebeCfg variable was added, it can overwritten with a value like:

```yaml
   zeebeCfg: |

    [data]
    snapshotPeriod = 5m
```

closes #14 

@salaboy would be nice if you could take a look and say if this makes sense or not.

### CODE REVIEW

**Comment by salaboy** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **salaboy**

If they are not already assigned, you can assign the PR to them by writing `/assign @salaboy` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=zeebe-io%2Fzeebe-cluster-helm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/zeebe-io/zeebe-cluster-helm/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: zeebe-cluster/templates/configmap.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ data:
     
     exec /usr/local/zeebe/bin/broker
   zeebe.cfg.toml: |
+    {{ .Values.zeebeCfg }}
```
Comment (salaboy) [inline zeebe-cluster/templates/configmap.yaml]: @Zelldon this makes sense based on our discussions... I really don't like that kind of entire file injection.. but I am ok with this for now.

File: zeebe-cluster/values.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ partitionCount: "3"
 replicationFactor: "3"
 cpuThreadCount: "2"
 ioThreadCount: "2"
+zeebeCfg: null
```
Comment (salaboy) [inline zeebe-cluster/values.yaml]: @Zelldon I don't think that we need this line. I think that we can add some checks in the `configmap.yaml` file to make sure that if .Values.zeebeCfg is not present anything breaks... 
Let's try to keep the values files as simple as possible.

Comment (Zelldon) [reply zeebe-cluster/values.yaml]: Yep we can remove it was not 100% sure if we need to declare this here in order to use it :man_shrugging: sorry I'm total beginner :D

Comment (salaboy) [reply zeebe-cluster/values.yaml]: @Zelldon no worries at all.. I would appreciate a PR to `github.com/zeebe-io/zeebe-helm`.. with some dev instructions on how to test local changes to the chart.. something that I am running here to check issues is `helm lint` inside the chart directory to at least make sure that the chart is not broken.

Comment (Zelldon) [reply zeebe-cluster/values.yaml]: I used `helm install . --dry-run --debug` as you mentioned to me

**Comment by Zelldon** [comment]
@salaboy i updated the PR

**Pull request merged**